drivers/volume/csi/csi.go

```diff
@@ -747,22 +965,21 @@ func (c *csi) recreateSnapshotForDeletion(
 	if err != nil {
 		return fmt.Errorf("failed to restore VolumeSnapshotClass for deletion: %s", err.Error())
 	}
-	log.ApplicationBackupLog(backup).Debugf("created volume snapshot class %s for backup %s deletion", vs.Name, snapshotID)
+	//log.ApplicationBackupLog(backup).Debugf("created volume snapshot class %s for backup %s deletion", vs.Name, snapshotID)
```

Comment:
any reason for commenting these out? I think we should uncomment these

Reply:
Commented it by mistake.